dev/tests/api-functional/testsuite/Magento/GiftMessage/Api/OrderItemGetRepositoryTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\GiftMessage\Api;
+
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class OrderItemGetRepositoryTest extends WebapiAbstract
+{
+    const SERVICE_VERSION = 'V1';
+    const SERVICE_NAME = 'giftMessageItemRepositoryV1';
+    const RESOURCE_PATH = '/V1/orders/items/';
```

Comment:
Such constant actually reduces test readability, please get rid of it (both cases).